bcipy/helpers/tests/test_copy_phrase_wrapper.py

```diff
@@ -0,0 +1,242 @@
+            ("E", 2.833274284028448),
+        ]
+        target_info = [
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+        ]
+
+        letters, timings, labels = cp.letter_info(triggers, target_info)
+        expected_letters = ["H", "D", "J", "B", "C", "A", "F", "G", "I", "E"]
+        expected_time = [
+            0.5670222830376588,
+            0.8171830819919705,
+            1.0843321380089037,
+            1.3329724550130777,
+            1.5825864360085689,
+            1.833380013005808,
+            2.083211077027954,
+            2.333359022042714,
+            2.583265081048012,
+            2.833274284028448,
+        ]
+        self.assertEqual(expected_letters, letters)
+        self.assertEqual(expected_time, timings)
+        self.assertEqual(len(letters), len(labels))
+
+        triggers = [
+            ("calibration_trigger", 0.0),
+            ("+", 0.1),
+            ("H", 0.5670222830376588),
+            ("D", 0.8171830819919705),
+            ("J", 1.0843321380089037),
+            ("B", 1.3329724550130777),
+            ("C", 1.5825864360085689),
+            ("A", 1.833380013005808),
+            ("F", 2.083211077027954),
+            ("G", 2.333359022042714),
+            ("I", 2.583265081048012),
+            ("E", 2.833274284028448),
+        ]
+        target_info = [
+            "calib",
+            "fixation",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+            "nontarget",
+        ]
+        letters, timings, labels = cp.letter_info(triggers, target_info)
+        self.assertEqual(expected_letters, letters)
+        self.assertEqual(expected_time, timings)
+        self.assertEqual(["nontarget"] * (len(letters)), labels)
+
+        # Test it throws an exception when letter is outside alphabet
+        with self.assertRaises(Exception):
+            cp.letter_info([("A", 0.0), ("*", 1.0)], ["nontarget", "nontarget"])
+
+    def test_init_series_evaluate_inquiry(self):
+        # TODO - it would be just as easy to store a snippet of real data, which would be a much better test
+        # (since this fake data likely does not exercise the dynamic range of the model)
+        alp = alphabet()
+
+        # Create fake data to provide as the user's response
+        # Create test items that resemble the fake training data, testing inference
+        # TODO - is this duration calculated precisely, or is it just large enough by coincidence?
+        # int(self.num_trial * self.device_spec.sample_rate * (self.params["time_flash"] + self.params["static_trigger_offset"]))
```

Comment:
I think we forgot to add self.params['trial_length'] here.

Reply:
Ok great - `num_trial * sample_rate * trial_length` works!